docs/prompt_templates.md

```diff
@@ -1,5 +1,6 @@
 # Prompt Templates
-Before evaluation, you can choose suitable prompt template for your model.
+
+Before evaluation, you can choose suitable prompt template for your model. Prompts can be referred to by version numbers (like `0.0`) or by short names (like `custom`). You can check the mapping [here](lm_eval/prompts.py).
```

Comment:
nit: for the links to work in GitHub, I think we need either the following
-  `../lm_eval/prompts.py` (relative path), or
- `/lm_eval/prompts.py` (absolute path)

Reply:
Thanks, should be fixed now.